### PR TITLE
add proxmox (vma) image format

### PR DESCRIPTION
**Pull request by illustris**
https://github.com/NixOS/nixpkgs/pull/144013

### CODE REVIEW

**Comment by Lassulus** [comment]
cool, can you add the format also to the README?

**Comment by illustris** [comment]
added

File: README.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ kexec-bundle | same as before, but it's just an executable
 lxc | create a tarball which is importable as an lxc container, use together with lxc-metadata
 lxc-metadata | the necessary metadata for the lxc image to start, usage: lxc image import $(nixos-generate -f lxc-metadata) $(nixos-generate -f lxc)
 openstack | qcow2 image for openstack
+proxmox | [VMA}(https://pve.proxmox.com/wiki/VMA) file for proxmox
```
Comment (06kellyjac) [inline README.md:47]: @Lassulus Should be `proxmox | [VMA](https://pve.proxmox.com/wiki/VMA) file for proxmox` (closing `]` after VMA)

image

Comment (Lassulus) [reply README.md:47]: whoops

**Pull request merged**